tests/core/config/test_data_node_config.py

```diff
@@ -142,8 +142,8 @@ def test_block_datanode_config_update_in_development_mode():
     assert Config.data_nodes[data_node_id].default_path == "foo.p"
     assert Config.data_nodes[data_node_id].storage_type == "pickle"
     assert Config.data_nodes[data_node_id].scope == Scope.SCENARIO
-    assert Config.data_nodes[data_node_id].cacheable == False
-    assert Config.data_nodes[data_node_id].properties == {"default_path": "foo.p"}
+    assert Config.data_nodes[data_node_id].cacheable is False
+    assert Config.data_nodes[data_node_id].properties == {"cacheable": False, "default_path": "foo.p"}
```

Comment:
I believe this should be skippable instead of cacheable right?

Reply:
Actually, I wanted to test that a user can still use "cacheable" as a custom property in data node configs.

Reply:
ah i see, I'm all good with that :D though I'd suggest a different test for it, but it's nothing major, just personal preference

Reply:
It should be on a separate test. You are right.